language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/CommonUtil.java

```diff
@@ -307,9 +307,10 @@ public static boolean isUnionOfType(TypeSymbol typeSymbol, TypeDescKind typeDesc
      * @param pkg {@link Package} package info to evaluate
      * @return {@link String} label computed
      */
-    public static String getPackageLabel(LSPackageLoader.PackageInfo pkg) {
+    public static String getPackageLabel(LSPackageLoader.ModuleInfo pkg) {
         String orgName = "";
-        if (pkg.packageOrg().value() != null && !pkg.packageOrg().value().equals(Names.ANON_ORG.getValue())) {
+        if (!pkg.packageOrg().value().isEmpty() && pkg.packageOrg().value() != null
```

Comment:
We don't need the null check here. If `pkg.packageOrg().value()` was null, it would have been an NPE already.